clang/unittests/Tooling/DependencyScanning/DependencyScannerTest.cpp

```diff
@@ -235,9 +235,11 @@ TEST(DependencyScanner, ScanDepsWithFS) {
   VFS->addFile(TestPath, 0,
                llvm::MemoryBuffer::getMemBuffer("#include \"header.h\"\n"));
 
-  DependencyScanningService Service(ScanningMode::DependencyDirectivesScan,
-                                    ScanningOutputFormat::Make, CASOptions(),
-                                    nullptr, nullptr, nullptr);
+  DependencyScanningService Service(
+      ScanningMode::DependencyDirectivesScan, ScanningOutputFormat::Make,
+      CASOptions(), nullptr, nullptr, nullptr, ScanningOptimizations::Default,
+      /*EagerLoadModules=*/false,
+      /*TraceVFS=*/false, llvm::sys::toTimeT(std::chrono::system_clock::now()));
```

Comment:
Why is this change necessary? It doesn't serve an explicit override of the negative stat caching option.